### PR TITLE
Simplify interface for calculation perf rates

### PR DESCRIPTION
**Pull request by akva2**
Minor refactor, introduces a struct and passes an instance of this instead of several separate variables.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/wells/PerforationData.hpp [hunk 1]
```diff
@@ -35,6 +35,14 @@ struct PerforationData
     std::size_t ecl_index;
 };
 
+struct PerforationRates
+{
+    double dis_gas{};
```
Comment (atgeirr) [inline opm/simulators/wells/PerforationData.hpp:40]: Will this zero-initialize? Even if it does, would it not be preferable to write it as `member = 0.0` to make it clearer?

**Review by atgeirr — APPROVED** [review]
Looks fine, only one question about initializing.

File: opm/simulators/wells/PerforationData.hpp (hunk 1 above)
Comment (akva2) [reply opm/simulators/wells/PerforationData.hpp]: it does zero initialize but sure, i can use more familiar syntax.

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/wells/PerforationData.hpp (hunk 1 above)
Comment (bska) [reply opm/simulators/wells/PerforationData.hpp]: As @akva2 correctly states, [list initialisation](https://en.cppreference.com/w/cpp/language/list_initialization) will perform [value initialisation](https://en.cppreference.com/w/cpp/language/value_initialization) in this case which, for built-in arithmetic types, will assign the type's zero value.

**Review by bska — APPROVED** [review]
I like it and I'll merge into master.

**Pull request merged**